Index: application/balancerApi/src/main/java/io/swagger/client/ApiClient.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- application/balancerApi/src/main/java/io/swagger/client/ApiClient.java	(revision 753f66a6aaa6426f7ddef405c5519693968f40be)
+++ application/balancerApi/src/main/java/io/swagger/client/ApiClient.java	(date 1657485258710)
@@ -13,6 +13,7 @@

 package io.swagger.client;

+import com.google.gson.JsonSyntaxException;
 import com.squareup.okhttp.*;
 import com.squareup.okhttp.internal.http.HttpMethod;
 import com.squareup.okhttp.logging.HttpLoggingInterceptor;
@@ -884,6 +885,13 @@
                 } catch (ApiException e) {
                     callback.onFailure(e, response.code(), response.headers().toMultimap());
                     return;
+                } catch (JsonSyntaxException e) {
+                    callback.onFailure(
+                            new ApiException(e.getClass().getName() + ": " + e.getMessage()),
+                            response.code(),
+                            response.headers().toMultimap()
+                    );
+                    return;
                 }
                 callback.onSuccess(result, response.code(), response.headers().toMultimap());
             }
Index: application/balancerApi/src/main/java/io/swagger/client/Version.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- application/balancerApi/src/main/java/io/swagger/client/Version.java	(date 1657485242846)
+++ application/balancerApi/src/main/java/io/swagger/client/Version.java	(date 1657485242846)
@@ -0,0 +1,5 @@
+package io.swagger.client;
+
+public class Version {
+    public static String VERSION = "0.1.0";
+}
