Index: swaggerApi/AUTOBUILD/javaBalancerClient/src/main/java/io/swagger/client/ApiClient.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- swaggerApi/AUTOBUILD/javaBalancerClient/src/main/java/io/swagger/client/ApiClient.java	(revision 2be1e06758057a4b4a45396d8e3b6ecb8bbe02f9)
+++ swaggerApi/AUTOBUILD/javaBalancerClient/src/main/java/io/swagger/client/ApiClient.java	(revision a94e5cd52c1644b1a4d40686a1b0d1dda2bac523)
@@ -12,6 +12,7 @@
 
 package io.swagger.client;
 
+import com.google.gson.JsonSyntaxException;
 import com.squareup.okhttp.*;
 import com.squareup.okhttp.internal.http.HttpMethod;
 import com.squareup.okhttp.logging.HttpLoggingInterceptor;
@@ -881,6 +882,13 @@
                 } catch (ApiException e) {
                     callback.onFailure(e, response.code(), response.headers().toMultimap());
                     return;
+                } catch (JsonSyntaxException e) {
+                    callback.onFailure(
+                            new ApiException(e.getClass().getName() + ": " + e.getMessage()),
+                            response.code(),
+                            response.headers().toMultimap()
+                    );
+                    return;
                 }
                 callback.onSuccess(result, response.code(), response.headers().toMultimap());
             }
Index: swaggerApi/AUTOBUILD/javaBalancerClient/src/main/java/io/swagger/client/Version.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- swaggerApi/AUTOBUILD/javaBalancerClient/src/main/java/io/swagger/client/Version.java	(revision a94e5cd52c1644b1a4d40686a1b0d1dda2bac523)
+++ swaggerApi/AUTOBUILD/javaBalancerClient/src/main/java/io/swagger/client/Version.java	(revision a94e5cd52c1644b1a4d40686a1b0d1dda2bac523)
@@ -0,0 +1,5 @@
+package io.swagger.client;
+
+public class Version {
+    public static String VERSION = "0.1.0";
+}
