Index: application/balancerApi/src/main/java/io/swagger/client/ApiClient.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- application/balancerApi/src/main/java/io/swagger/client/ApiClient.java	(revision 9a9e4e05daedc18abcad94e603ff2c9af0b09158)
+++ application/balancerApi/src/main/java/io/swagger/client/ApiClient.java	(date 1658233299263)
@@ -13,6 +13,7 @@

 package io.swagger.client;

+import com.google.gson.JsonSyntaxException;
 import com.squareup.okhttp.*;
 import com.squareup.okhttp.internal.http.HttpMethod;
 import com.squareup.okhttp.logging.HttpLoggingInterceptor;
@@ -883,6 +884,13 @@
                 } catch (ApiException e) {
                     callback.onFailure(e, response.code(), response.headers().toMultimap());
                     return;
+                } catch (JsonSyntaxException e) {
+                    callback.onFailure(
+                            new ApiException(e.getClass().getName() + ": " + e.getMessage()),
+                            response.code(),
+                            response.headers().toMultimap()
+                    );
+                    return;
                 }
                 callback.onSuccess(result, response.code(), response.headers().toMultimap());
             }
